### PR TITLE
V0.1.4 Release

### LINKED ISSUE #14: AB#12396 - Feature Request: compare_measures should accept more than one data input
The current version will compare any number of models but all models must use the same dataset for prediction. It would be useful to compare across models with different data inputs.

### LINKED ISSUE #15: AB#12408 - Bug: NaN values present in test_data cause compare_measures failure d.t. consistency score validation
AIF360's consistency score forces all data values to be finite, raising an error if non-finite values are present. Ideal behavior should skip measures that fail.

### LINKED ISSUE #18: AB#12466 - Bug: current flag_suspicious definition preventing some rows from being highlighted.
Lambda function uses x.iloc[0], seems to miss some out-of-range values. Check against the i-th entry via for loop instead.

### LINKED ISSUE #20: Feature Request: measure multiple protected attributes in iteration
It would be useful for the tool to accept one array of multiple protected attributes and iterate to produce outputs for each of those attributes.

See also Issue#19: "Feature Request: measure multiple protected attributes in groups"

### LINKED ISSUE #25: AB#12971 Installation failure for some users when installing from local copy of repo
On some machines installation will fail when installing from a local copy of the repo. The current suggsted workaround is as follows:

1. Download the repo from github
2. Open setup.py and replace all “fairMLHealth” with “fairmlhealth”
3. Install with “pip install <path to the downloaded folder>
4. import with “import fairmlhealth”

### PR DESCRIPTION
**Pull request by camagallen**
## New Features
- Additional validations and user feedback for the model_comparison methods
- Model_comparison now accepts multiple data inputs (one per model), in addition to the previous method allowing a single dataset to be used for all models
- New unit tests 
- Code style and hygiene updates in relation to the above

## Issues Addressed
Resolves #14, resolves #15, resolves #18, resolves #20, resolves #25

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,13 @@ All notable changes to this project will be documented in this file. Please do f
 
 _“Yesterday I was clever, so I wanted to change the world. Today I am wise, so I am changing myself.”_ - Rumi
 
+## [0.1.4] - 2020-12-09
```
Comment (kenviku) [inline CHANGELOG.md:7]: nit: Can you please add what was in 0.1.3?

**Review by kenviku — COMMENTED** [review]
LGTM. Please update the CHANGELOG to include details of 0.1.3.

File: CHANGELOG.md (hunk 1 above)
Comment (camagallen) [reply CHANGELOG.md:7]: good catch!

**Review by kenviku — APPROVED** [review]
🚀

**Pull request merged**